Dockerfile

```diff
@@ -2,10 +2,16 @@ FROM php:7.0.2-fpm
 
 COPY config/custom.ini /usr/local/etc/php/conf.d/
 
-RUN apt-get update && apt-get install -y zlib1g-dev libicu-dev libpq-dev \
+RUN apt-get update && apt-get install -y zlib1g-dev libicu-dev libpq-dev git libmemcached-dev \
     && docker-php-ext-install opcache \
     && docker-php-ext-install intl \
     && docker-php-ext-install pdo_mysql \
     && docker-php-ext-install pdo_pgsql \
+    ## APCu
     && pecl install apcu \
     && docker-php-ext-enable apcu
+    # For now we need to compile Memcached extension against PHP7 branch
+    && git clone --branch php7 https://github.com/php-memcached-dev/php-memcached /usr/src/php/ext/memcached \
```

Comment:
I got the code on Docker with many people asking this. It appears that no version has been released with PHP7 yet.